sdk/storage/Azure.Storage.Blobs/Changelog.txt

```diff
@@ -1,6 +1,14 @@
 Release History
 ===============
 
+12.0.0-preview.3 (2019-09)
+--------------------------
+- New Storage features for service version 2019-02-02 (including Customer
+  Provided encryption, expanded Set Tier support, the ability to set
```

Comment:
Maybe "Customer Provided Key" instead of "Customer Provided Encryption"

Reply:
Changed